src/components/Chat/Chat.tsx

```diff
@@ -24,6 +24,7 @@ export interface ChatProps {
   disableSse?: boolean;
   accessibility?: TockAccessibility;
   localStorageHistory?: TockLocalStorage;
+  autoCompletionEndPoint?: string;
```

Comment:
This should go in `TockSettings` rather than in `Chat`'s props (see #152 for an example).